packaged/lisk_snapshot.sh

```diff
@@ -174,7 +174,7 @@ echo -e "\\n$(now) Beginning snapshot verification process"
 bash lisk.sh start -p "$PM2_CONFIG"
 
 MINUTES=0
-until [[ ! $(pm2 info lisk.snapshot | grep status | awk '{print $4}') == "stopped" ]];  do
+until [[ ! $(pm2 jlist | jq -rc '.[] | select(.name | contains("lisk.snapshot")) | .pm2_env.status' == "stopped" ]];  do
```

Comment:
suggestions:
 - do not negate, check for `status == "running"` instead
 - use long options (i.e. `jq --raw-output`)
 - check for `.name == "lisk.snapshot` instead of using `contains()` -- unless there is a reason not to?